gradle.properties

```diff
@@ -2,6 +2,7 @@ fiatVersion=1.42.0
 korkVersion=7.204.0
 kotlinVersion=1.5.32
 org.gradle.parallel=true
+org.gradle.jvmargs=-Xmx6144m
```

Comment:
nit: I'd use what we're telling [github actions to use](https://github.com/spinnaker/orca/blob/7a95d5a7446329dd2fa99ba67019b5077c7760ac/.github/workflows/build.yml#L10)
```
-Xmx6g -Xms6g
```